webrender/src/renderer.rs

```diff
@@ -659,6 +662,13 @@ impl Renderer {
                                  options.precache_shaders)
         };
 
+        let ps_image_rect = try!{
```

Comment:
I add a new shader for textureRect. We could also merge this back to the original image shader. I don't like too much "#ifdef" in the code.